app/models/account_migration.rb

```diff
@@ -33,6 +45,17 @@ def perform!
 
       update_all_references
     end
+
+    dispatch if locally_initiated?
+  end
+
+  # TODO: docs
+  def subscribers
+    raise "You shouldn't federate AccountMigration if it was remotely initiated" if remotely_initiated?
```

Comment:
I don't think we need this. Our code will never do this, and when somebody wants to trigger that manually with rails console, then a pod can only do it when having both private keys anyway, which is only when locally initiated. The old home pod has only the private key of the old user and can't create a valid message.